File: JavaScript/1929. Concatenation of Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getConcatenation = function(nums) {
    var a = [];
    for(var i = nums.length - 1;i >= 0 ;i--) a.unshift(nums[i]);
    for(var i = nums.length - 1;i >= 0 ;i--) a.unshift(nums[i]);
    return a;
};

File: Java/1929. Concatenation of Array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int l = nums.length;
        int[] ans = new int[l * 2];

        for(int i = 0; i < 2; i++){
            for(int j = 0; j < l; j++){
                ans[i * l + j] = nums[j];
            }
        }

        return ans;
    }
}
